src/resources/org/apache/cassandra/graph/graph.html

```diff
@@ -238,7 +238,11 @@
                     return d.intervals[d.intervals.length-1][time_index];
                 }
             }) * 1.1 * 100) / 100;
-            query.ymin = ymin = query.ymin ? query.ymin : 0;
+            query.ymin = ymin = query.ymin ? query.ymin : Math.min(0, d3.min(data, function(d) {
+                return d3.min(d.intervals, function(i) {
+                    return getMetricValue(i);
+                });
+            }) * 1.1 * 100) /100;
```

Comment:
This path does not include rounding to the nearest hundredth, the point of which is to make the numbers in the fields look a little better. Either add it, or remove the multiplication and division by 100 which do not serve any purpose without `Math.round`.